### PR TITLE
Create organisational-leadership.md

### PR DESCRIPTION
**Pull request by pherterich**
First addition to #678 - @NPDebs can you please make this a proper page (copying from consulting is fine and putting it somehow under services)

### CODE REVIEW

**Comment by pherterich** [comment]
@yochannah - any changes to the Google Doc should be copied across to here

**Comment by NPDebs** [comment]
Looks good, Patri! I'll also review for any formatting that might be necessary.

But first...I was going to clarify something from @yochannah.

We want to have:
- one [consulting page](https://openlifesci.org/consulting.html) that contains all the services in little cards, and
- individual pages with a detailed description of the services? (With this pattern of URL _we-are-ols.org/consulting/name-of-service.md_

Yes? 😃

**Comment by yochannah** [comment]
@NPDebs yes, exactly! Please make tweaks as needed to make sure this conforms

**Review by NPDebs — COMMENTED** [review]
Hi Patri. 👋
I formatted the text a little and added an image to the page (the same image in the `/consulting.md` page). Also made a tiny suggestion which you're free to ignore. 😆

File: organisational-leadership.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+##  What to expect when you’re setting up a new value-driven organisation
+* Reflection worksheet (have you thought about… x).
+* Guided two-hour workshop with 8 follow-up 1:1 coaching sessions.
+  
+### What is covered: 
+* **Real-world practicalities:**
+  * Preparing to receive money.
+  * Creating job ads and hiring equitably. 
+  * Pensions, benefits, vacation days and leave policies.
+  * Identifying necessary support, such as hiring accounting and legal.
```
[the remark this answers is not in the document]
Comment (pherterich) [reply organisational-leadership.md]: Added :)

**Comment by pherterich** [comment]
@yochannah do you think you can officially approve this?

**Review by yochannah — APPROVED** [review]

**Pull request merged**